### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by Xero92** [ForlaxPy/openbullet-1.2#19]
Hi,

Thanks for your amazing work so far, I really appreciate it.
But I got a suggestion for the config section.
Sorry for posting it here, but I couldn't find where to post suggestions 

1. Add search option when selecting a config [https://i.imgur.com/RYLRjEz.png] [instead of using Configs menu]

**Comment by shdf2** [comment]
yes. at least sorting by name, by default would also be more handy.

**Comment by Xero92** [comment]
> Nice idea I will look forward to add it in the upcoming updates

Also, how about full .GIF support for the background theme image ?
Or maybe even allow to play video as background :)

**Comment by Xero92** [comment]
Well? :P
